asana-helpers-markdown.js

```diff
@@ -1,10 +1,10 @@
 // ==UserScript==
 // @name         Asana Helpers - Markdown, Expand Comments, Theme Switch
 // @namespace    https://github.com/jpbochi/user-scripts
-// @version      1.3.1
+// @version      1.3.0
 // @description  Adds 3 helper buttons, plus paste in markdown format.
 // @author       Nick Goossens, JP Bochi, Karl K
-// @match        *://app.asana.com/*
+// @match        https://app.asana.com/*
```

Comment:
Looks like you're reverting a small change I did on https://github.com/jpbochi/user-scripts/pull/5

```suggestion
// @match        *://app.asana.com/*
```